app/scripts/metamask-controller.js

```diff
@@ -4996,11 +4996,11 @@ export default class MetamaskController extends EventEmitter {
     if (sender.url) {
       if (this.onboardingController.store.getState().completedOnboarding) {
         if (this.preferencesController.store.getState().usePhishDetect) {
-          const { hostname } = new URL(sender.url);
           this.phishingController.maybeUpdateState();
           // Check if new connection is blocked if phishing detection is on
-          const phishingTestResponse = this.phishingController.test(hostname);
+          const phishingTestResponse = this.phishingController.test(sender.url);
```

Comment:
@Gudahtt, the `test` method in `phishingDetector` was updated to accept a URL in this [PR](https://github.com/MetaMask/core/pull/4465) [see change](https://github.com/MetaMask/core/pull/4465/files#diff-a3e0475a00b3bf152d951e242c271df7621313a1cbc20673c1bb882f935bd740R147). However, I believe this is a breaking change that should have been mentioned in the README.